### PR TITLE
10.0 pos invoice postponed publication 2

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_invoice_postponed/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "images": ['images/postponed2.png'],
+    "version": "10.0.1.0.0",
+    "application": False,
+
+    "author": "IT-Projects LLC, Kolushov Alexandr",
+    "support": "apps@it-projects.info",
+    "website": "https://it-projects.info/team/KolushovAlexandr",
+    "license": "LGPL-3",
+    # "price": 9.00,
+    # "currency": "EUR",
```
Comment (yelizariev) [inline pos_invoice_postponed/__manifest__.py]: uncomment both and set price to 90. Use :euro: as a tag

**Comment by ilmir-k** [comment]
closes #480

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

File: pos_invoice_postponed/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": """Creation of Postponed invoices in POS""",
+    "summary": """This module allows the usage of a regular POS order payment process to create an invoice to be paid later""",
+    "category": "Point of Sale",
+    # "live_test_url": "http://apps.it-projects.info/shop/product/DEMO-URL?version=10.0",
+    "images": ['images/postponed2.png'],
```
Comment (ilmir-k) [inline pos_invoice_postponed/__manifest__.py:9]: @Mirgalimova 
it should be  ``"images": ['images/postponed2.jpg'],``

Comment (ilmir-k) [reply pos_invoice_postponed/__manifest__.py:9]: fixed here #993

File: pos_invoice_postponed/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      Thank you for understanding!
+
+      `IT-Projects Team <https://www.it-projects.info/team>`__
+
+Further information
+===================
+
+Demo: http://runbot.it-projects.info/demo/pos-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0//
```
Comment (ilmir-k) [inline pos_invoice_postponed/README.rst:41]: @Mirgalimova 
It should be ``HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_invoice_postponed``

Comment (ilmir-k) [reply pos_invoice_postponed/README.rst:41]: fixed here #993

**Pull request merged**